File: day_1_js_basics/fizzBuzz.js
const fizz=parseInt(process.argv[2]);
const buzz=parseInt(process.argv[3]);
for(i=1;i<=100;i++)
{
    if(i%fizz==0&&i%buzz==0)
    {
        console.log("fizzbuzz");
    }
    else if(i%fizz==0)
    {
        console.log("fizz");
    }
    else if(i%buzz==0)
    {
        console.log("buzz");
    }
    else    console.log(i);
}

File: day_35_rails_object/FizzBuzz.rb
class FizzBuzz
    attr_accessor :first_number, :second_number
    def initialize(first_number, second_number)
      @first_number = first_number
      @second_number = second_number
    end

    def run
        arr = []
        for value in 1..100 do
            if value % first_number == 0
                arr.push('fizz')
            elsif value % second_number == 0
                arr.push('buzz')
            elsif value % second_number  == 0   
                arr.push('fizzbuzz')
            else
                arr.push(value)
            end
        end
        print arr
    end
    
end